packages/server/test/support/helpers/e2e.coffee

```diff
@@ -57,19 +57,25 @@ replaceDurationSeconds = (str, p1, p2, p3, p4) ->
 
 replaceDurationFromReporter = (str, p1, p2, p3) ->
   ## duration='1589'
-
   p1 + _.padEnd("X", p2.length, "X") + p3
 
 replaceNodeVersion = (str, p1, p2, p3) ->
-  ## Node Version: 1.2.3
-
+  ## Node Version: 10.2.3
   return p1 + _.padEnd("X", p2.length, "X") + ' (/foo/bar/node)' + ' '.repeat(p3.length - 16)
```

Comment:
doesn't this make the replacement text size depend on the underlying node version?

Node 12.0.0 will become Node XXXXXX, Node 12.11.0 will become XXXXXXX

probably not worth too much time to fix since there's only 1 or 2 tests that use custom Node, but it will come up whenever the Node version string on CI changes length.

Reply:
Yes, I know 😞 I didn't want to make anymore changes. 😓 I'll fix it

Reply:
i think `1_system_node_spec` will be the only snapshot that changes